src/main/java/ledger/database/storage/ISQLiteDatabaseNote.java

```diff
@@ -58,14 +62,14 @@ default void editNote(Note note) {
     default List<Note> getAllNotes() throws StorageException {
         try {
             Statement stmt = getDatabase().createStatement();
-            ResultSet rs = stmt.executeQuery("SELECT * FROM NOTE;");
+            ResultSet rs = stmt.executeQuery("SELECT * FROM " + NoteTable.TABLE_NAME + ";");
```

Comment:
I got rid of all occurrences of "SELECT *" in the ISQLiteDatabase____ files in my refactoring work on the f_2refactoring branch.